opset/configurator.py

```diff
@@ -559,3 +618,26 @@ def load_logging_config(
             logging.getLogger(logger_name).setLevel(min_level)
 
     return root_logger
+
+
+def get_opset_config() -> dict[str, Any] | None:
+    """Search for opset config and load it if it exists.
+
+    :return: Opset Config or None
```

Comment:
You have a mix of Sphinx and Google style docstrings, I don't remember which is used by Opset but it should probably the same everywhere

Reply:
Intellij auto complete madness :upside_down_face: